user_docs/en/userGuide.t2t

```diff
@@ -2549,8 +2549,10 @@ Add-ons may do any of the following:
 -
 
 NVDA's Add-on Store allows you to browse and manage add-on packages.
-Add-ons that are available in the Add-on Store are free.
-Other add-ons which require payment do exist, such as those which add support for commercial speech synthesizers; but these must be installed from external sources.
+All add-ons that are available in the Add-on Store are free.
+However, some free add-ons do require users to pay for a license or additional software before they can be used.

```

Comment:
Emphasize the fact that only the download is free of charge; also avoids the confusion between free of charge and free software licensing.

```suggestion
All add-ons that are available in the Add-on Store can be downloaded for free.
However, some of them may require users to pay for a license or additional software before they can be used.
```